app/enssubdomain/models.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models
+from django.utils import timezone
+from django.utils.timezone import localtime
+
+# Create your models here.
+from economy.models import SuperModel
+
+
+def get_time():
+    """Get the local time."""
+    return localtime(timezone.now())
+
+
+class ENSSubdomainRegistration(SuperModel):
+    # Defines the schema for storing ENS sub-domain registration
```

Comment:
Replace with a triple `"` docstring and modify to imperative mood: `"""Define ... registration."""`